Source/WebKit/Shared/AuxiliaryProcessCreationParameters.serialization.in

```diff
@@ -24,4 +24,6 @@ struct WebKit::AuxiliaryProcessCreationParameters {
     String wtfLoggingChannels;
     String webCoreLoggingChannels;
     String webKitLoggingChannels;
+    bool crashOnCoreIPCSecureCoding;
+    HashSet<String> classNamesExemptFromSecureCodingCrash;
```

Comment:
Couldn't these be combined to a std::optional<HashSet<String>>?  It wouldn't make sense to have false then a non-empty set.